File: 2021/11/18/main.go
package main

func findDisappearedNumbers(nums []int) []int {
	l := make(map[int]int, len(nums)+1)
	for i := 1; i <= len(nums); i++ {
		l[i] = 0
	}
	for _, n := range nums {
		l[n] = 1
	}
	r := []int{}
	for i := 1; i <= len(nums); i++ {
		if l[i] == 0 {
			r = append(r, i)
		}
	}
	return r
}

File: 2021/11/18/main.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        l = {i: 0 for i in range(1, len(nums)+1)}
        for n in nums:
            l[n] = 1
        r = []
        for i in range(1, len(nums)+1):
            if l[i] == 0:
                r.append(i)
        return r